configs/embargo-sia.yaml

```diff
@@ -0,0 +1,71 @@
+facility_name: Rubin-LSST
```

Comment:
The formal name is `Rubin:Simonyi`.  Since this repo has LATISS and LSSTComCam in it (and LSSTCam) there will also need to be a facility_map section mapping instrument to facility (Rubin:1.2m for LATISS and Rubin:Simonyi for LSSTCam and LSSTComCam)